content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,212 @@
+
+1. Install Istio using the operator with the default profile:
+
+    {{< text bash >}}
+    $ kubectl apply -f https://preliminary.istio.io/operator.yaml
+    {{< /text >}}
+
+1. (Optionally) change profiles from the default profile to one of the following profiles:
+
+{{< tabset cookie-name="profile" >}}
+
+{{< tab name="permissive mutual TLS" cookie-value="permissive" >}}
+When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
+mutual TLS traffic. Clients send plaintext traffic unless configured for
+[mutual TLS migration](/docs/tasks/security/mtls-migration/).
+
+Choose this profile for:
+
+* Clusters with existing applications, or
+* Applications where services with an Istio sidecar need to be able to
+  communicate with other non-Istio Kubernetes services
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-demo.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="strict mutual TLS" cookie-value="strict" >}}
+This profile will enforce
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) between all clients and servers.
+
+Use this profile only on a fresh Kubernetes cluster where all workloads will be Istio-enabled.
+All newly deployed workloads will have Istio sidecars installed.
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-demo-auth.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="SDS" cookie-value="sds" >}}
+This profile enables
+[Secret Discovery Service](/docs/tasks/security/auth-sds) between all clients and servers.
+
+Use this profile to enhance startup performance of services in the Kubernetes cluster. Additionally
+improve security as Kubernetes secrets that contain known
+[risks](https://kubernetes.io/docs/concepts/configuration/secret/#risks) are not used.
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-sds.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="default" cookie-value="default" >}}
+This profile enables Istio's default settings. This profile is installed during the first step.
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-default.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="minimal" cookie-value="minimal" >}}
+This profile deploys a Istio's minimum components to function.
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-minimal.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Verifying the installation
+
+{{< warning >}}
+This document is a work in progress. Expect verification steps for each of the profiles to
```

Comment:
make sure we remove this before 1.4 ships

Reply:
yes once we have examples of verirfication they will be integrated in the docs.  from my understanding profile switching just became functional a few hours ago.